src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,72 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
+    private CinemaHall cinemaHall;
```

Comment:
It's good to add JoinColumn everywhere in ManyToOne relations. 

Also you can think about cascades (Perhaps if we delete a film, then it would be good to delete the sessions, which would not exist without them.) It's not obligatory, more to think from business logic's side

Reply:
Yes, I missed this moment (about joincolumns)
since we have a project diagram and bidirectional relations is not indicated on it I leave it as it is (but yes, if the diagram shows bidirectional relations, a cascade is possible).